### PR TITLE
:bug: Fixes closing tags

### PR DESCRIPTION
**Pull request by jmelberg-okta**
## Description:
- Fixes a variety of missing closing `<span>` tags by updating to `api_operation` tags.

## Type of Pull Request:

- [ ] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [X] Functional (CSS changes, JS updates, or layout modifications)

## How Has This Been Tested?
- [X] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

### CODE REVIEW

**Comment by jmelberg-okta** [comment]
For your review: @mystiberry-okta @rchild-okta

File: _source/_docs/api/resources/factor_admin.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ deactivate         | Denies use of this factor for MFA
 ### Get Org Factor
 {:.api .api-operation}
 
-<span class="api-uri-template api-uri-get"><span class="api-label">GET</span> /api/v1/org/factors
+<span class="api-uri-template api-uri-get"><span class="api-label">GET</span> /api/v1/org/factors</span>
```
Comment (rchild-okta) [inline _source/_docs/api/resources/factor_admin.md]: Now that I'm looking at it, we should probably just switch these out with the [api_operation tag](https://github.com/okta/okta.github.io/blob/master/_source/_plugins/ApiTags.rb#L27-L38). You can see some example of using it here:
https://github.com/okta/okta.github.io/search?utf8=%E2%9C%93&q=api_operation&type=

Comment (mystiberry-okta) [inline _source/_docs/api/resources/factor_admin.md]: There's a lot of these to clean up, fyi. Not just this file.

File: _source/_code/dotnet/powershell.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: software
 title: Okta API PowerShell SDK
-excerpt: PowerShell bindings for the Okta API. <a href="https://github.com/okta/oktasdk-csharp/tree/master/Okta.Core.Automation/">Get started now</a>.
-github_url: https://github.com/okta/oktasdk-csharp
+excerpt: PowerShell bindings for the Okta API. <a href="https://github.com/okta/okta-sdk-csharp/tree/master/Okta.Core.Automation/">Get started now</a>.
+github_url: https://github.com/okta/okta-sdk-csharp
```
Comment (rchild-okta) [inline _source/_code/dotnet/powershell.md]: Hmm, is this ready yet? 404's for me

Comment (jmelberg-okta) [inline _source/_code/dotnet/powershell.md]: I was hoping this was going to be changed this week. I reverted the change to only include the python and java SDK names.

**Pull request merged**